build.gradle

```diff
@@ -315,6 +315,9 @@ configure(javaProjects) {
     spotbugs {
         excludeFilter = new File(project.parent.projectDir, "codequality/spotbugs/excludeFilter.xml")
     }
+    spotbugsTest.enabled = false
```

Comment:
I think there's value in keeping this. How much does it shave off the build time?

Reply:
On successful builds it's ~2 minutes on my local machine however i ran into multiple instances where despite increasing the heap size of the gradle JVM the spotbugs process would just get stuck infinitely before completely still blowing away the heap. For now I've turned it off for the test files. I tried upgrading spotbugs to the latest version but that ran into other plugin classpath issues. @chali was looking at that and maybe if he gets it resolved we can put this back in and see if it's still a problem